backend/utils.ts

```diff
@@ -43,15 +43,15 @@ export const mapToQueryString = (params: Record<string, unknown>): string =>
     })
     .join('&')
 
-export const toMap = <T, F extends keyof T>(items: T[] = [], name: F): Map<T[F], T> => {
-  return items.reduce((result, item) => {
+export const toMap = <T, F extends keyof T>(items: T[], name: F): Map<T[F], T> => {
+  return (items || []).reduce((result, item) => {
```

Comment:
new eslint rule does not like default arguments not appearing before the end of the parameter list